packages/package_info_plus/package_info_plus/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.4.3
+
+- Android: Update ffi dependency
```

Comment:
Is there a reason, the changelog says `Android:` while the changes are in `package_info_plus_windows`?

Reply:
It was mistake. I change to `Windows` instead of `Android`.